helpdesk_mgmt/data/helpdesk_data.xml

```diff
@@ -32,105 +32,53 @@
-                        <td align="center" style="min-width: 590px;">
-                          <table
-                                    width="590"
-                                    border="0"
-                                    cellpadding="0"
-                                    bgcolor="#875A7B"
-                                    style="min-width: 590px; background-color: rgb(135,90,123); padding: 20px; border-collapse:separate;"
-                                >
-                            <tr>
-                              <td
-                                            valign="middle"
-                                            align="left"
-                                            style="color: #fff; padding-top: 10px; padding-bottom: 10px; font-size: 12px;"
-                                        >
-                                <t t-out="object.company_id.phone" />
-                              </td>
-                              <td
-                                            valign="middle"
-                                            align="left"
-                                            style="color: #fff; padding-top: 10px; padding-bottom: 10px; font-size: 12px;"
-                                        >
-                                <t t-out="object.company_id.email" />
-                              </td>
-                              <td
-                                            valign="middle"
-                                            align="left"
-                                            style="color: #fff; padding-top: 10px; padding-bottom: 10px; font-size: 12px;"
-                                        >
-                                <t t-out="object.company_id.website" />
-                              </td>
-                            </tr>
-                          </table>
-                        </td>
-                      </tr>
-                    </tbody>
-                </table>
+                <div>
+                    <p>Hello ${object.partner_id.name or object.partner_name or ''},</p>
+                    <p>The ticket "${object.display_name or 'n/a'}" has been closed.</p>
+                    % if object.partner_can_access():
+                        <div style="margin: 16px 0px 16px 0px;">
+                            <a
+                            href="${object.get_access_link()}"
+                            style="background-color: #875A7B; padding: 8px 16px 8px 16px; text-decoration: none; color: #fff; border-radius: 5px; font-size:13px;"
+                        >
+                                View Ticket
+                            </a>
+                        </div>
+                    % endif
+                    Thank you,<br /><br />
+                    ${user.signature | safe}<br />
+                </div>
+            </field>
+        </record>
+        <record id="created_ticket_template" model="mail.template">
```

Comment:
This is a new template.

Reply:
Would you prefer its own commit/PR?

Reply:
Or its own PR explaining the reasons, so that not everything must be approved together. Anyway, this is not working, as the syntax is not v16. Did you check it?

Reply:
yes, I'm working on it.